components/booking/AvailableTimes.tsx

```diff
@@ -44,6 +45,12 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
     eventTypeId,
   });
 
+  const [brand, setBrand] = useState("#292929");
```

Comment:
i wonder if this is the best place to hard code it here. @zomars @emrysal thoughts?

Reply:
This is juts a fallback. I don't think this should be an issue but @zomars and @emrysal might confirm if there is a better alternative available.

Reply:
If it works is good enough for now 👍🏽